examples/icon-color.js

```diff
@@ -19,13 +19,18 @@ const london = new Feature({
 const madrid = new Feature({
   geometry: new Point(fromLonLat([-3.683333, 40.4])),
 });
+const paris = new Feature({
+  geometry: new Point(fromLonLat([2.353, 48.8566])),
+});
+const berlin = new Feature({
+  geometry: new Point(fromLonLat([13.3884, 52.5169])),
+});
 
 rome.setStyle(
   new Style({
     image: new Icon({
       color: '#8959A8',
       crossOrigin: 'anonymous',
-      imgSize: [20, 20],
```

Comment:
The imgSize is necessary for svg images in Internet Explorer 11

Reply:
+1, Thanks :-)